homeassistant/components/tautulli/sensor.py

```diff
@@ -53,12 +61,178 @@
     }
 )
 
-SENSOR_TYPES: tuple[SensorEntityDescription, ...] = (
-    SensorEntityDescription(
+
+def get_top_stats(coordinator: TautulliDataUpdateCoordinator, key: str) -> str | None:
+    """Get top statistics."""
+    value = None
+    for stat in coordinator.home_stats:
+        if stat.stat_id == key:
+            value = stat.rows[0].title if stat.rows else None
+        elif stat.stat_id == "top_users" and key == ATTR_TOP_USER:
+            value = stat.rows[0].user if stat.rows else None
+    return value
+
+
+@dataclass
+class TautulliSensorEntityDescription(SensorEntityDescription):
+    """Class to describe a Tautulli sensor."""
+
+    value: Callable[[TautulliDataUpdateCoordinator, str], Any] = lambda val, _: val
```

Comment:
```suggestion
    value_fn: Callable[[TautulliDataUpdateCoordinator, str], Any]
```

Reply:
how would this work without the lambda?

Reply:
You can do it like this <https://github.com/home-assistant/core/blob/dev/homeassistant/components/github/sensor.py#L25-L44>